src/transformStateWithClones.js

```diff
@@ -8,6 +8,30 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  let newState = { ...state };
```

Comment:
NewState is never reassigned. Use 'const' instead.

Reply:
It will be reassigned on 27 line